### PR TITLE
Add padding to pass rates script output

### PR DESCRIPTION
**Pull request by honza**
Sorry, my OCD just could not

### CODE REVIEW

**Comment by honza** [comment]
cc @derekhiggins

**Comment by dougsland** [comment]
looks sane to me.
/test e2e-metal-ipi-serial-ipv4

File: scripts/periodics_pass_rates.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```
Comment (zaneb) [inline scripts/periodics_pass_rates.sh:1]: Since we're doing OCD, this file definitely contains bash-specific syntax like `$()` :slightly_smiling_face:

Comment (honza) [reply scripts/periodics_pass_rates.sh:1]: It's not OCD; it literally didn't work on my machine :slightly_smiling_face:.  I'll change it to `#!/usr/bin/env bash`, would that be satisfactory to you?

**Pull request merged**